docs/source/getting_started/content/proc_configuring-the-virtual-machine.adoc

```diff
@@ -0,0 +1,50 @@
+[id="configuring-the-virtual-machine_{context}"]
+= Configuring the virtual machine
+
+The default number of vCPUs and amount of memory available to the {prod} virtual machine can be configured using the `cpus` and `memory` configurable properties, respectively.
+
+[IMPORTANT]
+====
+Changes to the configuration of the virtual machine require the creation of a new {prod} virtual machine.
+To create the new virtual machine, first delete the existing {prod} instance, then start a new instance with the configuration changes:
+
+[subs="+quotes,attributes"]
+----
+$ {bin} delete
+$ {bin} start
+----
+====
+
+[WARNING]
+====
+The [command]`{bin} delete` command will result in the loss of data stored in the {prod} virtual machine.
+Save any desired instance information before running this command.
+====
+
+.Procedure
+
+* To increase the number of vCPUs available to the virtual machine:
```

Comment:
I just had a conversation with Junko about this lead-in sentence. I prefer the way you wrote it here, but she told me that broken sentences like this require manual translator intervention, so it's better to have a complete sentence in lead-ins. The IBM Style Guide doesn't explicitly say that you have to have a complete sentence here, but its examples all include something like "complete the following step(s)" or in this case I'd write "enter the following command:"

Reply:
We've used "run the following command" in Minishift docs before, so I'm fine with that. This would require some changes across the entire documentation set, though, so it might be worth creating a new docs issue to encompass the changes across the entire set.

For what it's worth, I keep hearing contradictory things regarding wording such as this. The contradictions largely seem to come from minimalism telling us to remove the fluff, while localization benefits from complete sentences.

Reply:
I won't address this particular point in this PR simply because the requested change is larger in scope than this PR alone. For consistency, I'm okay with this PR going in as-is with the lead-in sentences. I'll create another issue to address lead-in sentences across the documentation set.

Reply:
Yeah, that's fine. We have the same issues in our docs. With anything like this that requires a lot of changes for the sake of consistency, it's usually more practical to start doing the "right" thing and then extend to legacy content in a separate phase of work.

Reply:
As far as your remark about minimalism...one thing that Ingrid mentioned is that minimalism does not necessarily mean fewer words. It means being as concise *and clear* as possible. In this case, I agree that the extra words are fluffy, and because of that, it was important for me to hear directly from Junko what the exact issues are. So minimalistic writing does need to take into account other issues, like translation. And sometimes clarity requires more words, but that doesn't meant that more words are fluff.

Reply:
I'm aware of the goal of minimalism, and that it doesn't mean fewer words. :)

In this particular context (lead-in sentences), though, I've explicitly heard advice going in opposing directions. ;)